docs/user-guide/src/SUMMARY.md

```diff
@@ -1,10 +1,26 @@
 # Metal3 Project
 
 [Introduction](introduction.md)
-[Concepts]()
 
 - [Project Overview]()
-- [Install Baremetal Operator]()
-- [Install Cluster-api-provider-metatl3]()
-
-[Community]()
\ No newline at end of file
+- [Baremetal Operator]()
```

Comment:
I think it would be useful to have a common initial paragraph (ie `[Introduction]`?) for each main section dedicated to briefly explain what is the current object and what's meant for

Reply:
@andfasano  Definitely. I was thinking that we could write it down in the chapter name. For example, for CAPM3 it would located inside `Cluster-api-provider-metal3` (in other words section4), for BMO in `Baremetal Operator` (section2) and etc.